src/main/java/org/littil/api/user/service/UserService.java

```diff
@@ -110,4 +124,18 @@ public void createAndPersistDevData(UUID id, String auth0id, String email) {
         user.setEmailAddress(email);
         this.repository.persist(user);
     }
+
+    /*
+     * Method can not be used in stream operation.
+     * This will cause a com.auth0.exception.RateLimitException.
+     * HttpStatus 429 (Too Many Requests) is returned by auth0.
```

Comment:
is this comment still relevant now we use resilience4j ?

Reply:
This comment is still relevant.